runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -159,11 +159,21 @@ public constructor(
     @ImplicitReflectionSerializer
     public inline fun <reified T : Any> fromJson(tree: JsonElement): T = fromJson(context.getContextualOrDefault(T::class), tree)
 
-    companion object : StringFormat {
-        @Suppress("DEPRECATION")
+    /**
+     * The default instance of [Json] in the form of companion object.
+     */
+    @UnstableDefault
+    public companion object Default : StringFormat {
+        private const val message =
+            "Top-level JSON instances are deprecated for removal in the favour of user-configured one. " +
+                    "You can either use a Json top-level object, configure your own instance  via 'Json {}' builder-like constructor, " +
+                    "'Json(JsonConfiguration)' constructor or by tweaking stable configuration 'Json(JsonConfiguration.Stable.copy(prettyPrint = true))'"
```

Comment:
`Stable` configuration should [probably] be deprecated and/or renamed to `PreRelease`, because once we're 1.0, `Default` should be stable and preferred for new users.

Reply:
Let's fix it separately